README.md

```diff
@@ -54,58 +60,99 @@ Local: http://localhost:3000
+| Multiplexed address   | [/search/MDZ464OWNGEL4X2DE6JPLEARO2WJ4AGCBN3XM7E4ZSLPHRBV6AZB6AAAAAAAAAAAAGW4M](https://steexp.com/search/MDZ464OWNGEL4X2DE6JPLEARO2WJ4AGCBN3XM7E4ZSLPHRBV6AZB6AAAAAAAAAAAAGW4M) |
+| Ledger                | [/search/10000000](https://steexp.com/search/10000000)                                                                                                                           |
+| Transaction           | [/search/26a568681712a44a515b2c90dcfaadb3ed2c40dc60254638407937bee4767071](https://steexp.com/search/26a568681712a44a515b2c90dcfaadb3ed2c40dc60254638407937bee4767071)           |
+| Asset Code            | [/search/NGN](https://steexp.com/search/NGN)                                                                                                                                     |
+| Anchor Name           | [/search/ripplefox](https://steexp.com/search/ripplefox)                                                                                                                         |
+| Anchor Name (Partial) | [/search/fox](https://steexp.com/search/fox)                                                                                                                                     |
 
 ### Misc
 
-| Resource    | URI |
-| ----------- | --- |
+| Resource    | URI                                                                                                                                                            |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Transaction | [/tx/26a568681712a44a515b2c90dcfaadb3ed2c40dc60254638407937bee4767071](https://steexp.com/tx/26a568681712a44a515b2c90dcfaadb3ed2c40dc60254638407937bee4767071) |
-| Ledger      | [/ledger/10000000](https://steexp.com/ledger/10000000) |
-| Anchor      | [/anchor/apay.io](https://steexp.com/anchor/apay.io) |
-| Asset       | [/asset/NGN](https://steexp.com/asset/NGN) |
+| Ledger      | [/ledger/10000000](https://steexp.com/ledger/10000000)                                                                                                         |
+| Anchor      | [/anchor/apay.io](https://steexp.com/anchor/apay.io)                                                                                                           |
+| Asset       | [/asset/NGN](https://steexp.com/asset/NGN)                                                                                                                     |
 
 ## Exploring Private / Local Development Networks<a name="private-networks"></a>
 
-steexp will connect to a local horizon instance at http://localhost:8000 by default. If your running a local private network for development this is quite handy for browsing your changes to the ledger.
```

Comment:
GD2VTBFF56BPQJOLNO3DZSMLDJMGC4GB2RFHGJ7N5RJGRODTTZ5ZHTDY

Reply:
3000

Reply:
Can I have the money right knaw

Reply:
Like